app/Controllers/EscolasController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ControleAlunos\Controllers;
+
+use PDO;
+use PDOException;
+use ControleAlunos\Models\Escola;
+use ControleAlunos\Repositories\EscolasRepository;
+use ControleAlunos\Controllers\AbstractController;
+
+class EscolasController extends AbstractController
+{
+    protected $escolasRepository;
+
+    protected function beforeAction()
+    {
+        $this->escolasRepository = new EscolasRepository($this->pdo);
+    }
+
+    public function index()
+    {
+        $escolas = $this->escolasRepository->all();
+        require_once __DIR__ . "/../../views/listagem-escolas.php";
+    }
+
+    public function create()
+    {
+        require_once __DIR__ . "/../../views/registro-escolas.php";
+    }
+
+    public function edit()
+    {
+        $escolas = $this->escolasRepository->all();
+        $escola = null;
+
+        require_once __DIR__ . "/../../views/atualizacao-escolas.php";
+    }
+
+    public function store()
+    {
+        $escola = new Escola();
+        $escola->nome = $_POST["nome"];
+        $escola->endereco = $_POST["endereco"];
+        $escola->status = $_POST["situacao"];
+
+        $this->escolasRepository->store($escola);
+
+        header("Location: listagem-escolas");
+        exit();
+    }
+
+    public function update()
+    {
+        if (isset($_POST["confirmar"]) && (!empty($_POST["escola_id"]))){
+            $escola = new Escola();
+            $escola->id = $_POST["escola_id"];
+            $escola->nome = $_POST["nome"];
+            $escola->endereco = $_POST["endereco"];
+            $escola->status = $_POST["situacao"];
+                
+            $this->escolasRepository->update($escola);
+
+            header("Location: listagem-escolas");
+            exit();
+        }
+
+        if (isset($_POST["excluir"]) && (!empty($_POST["escola_id"]))){
+            $this->escolasRepository->delete($_POST["escola_id"]);
+
+            header("Location: listagem-escolas");
+            exit();
+        }
+    }
+
+    public function show()
+    {
+        try {
+            if (isset($_POST["escola_id"])) {
```

Comment:
```suggestion
            if (isset($_GET["escola_id"])) {
```
Em REST o get é usado para resgatar registros.
Então o index e show, devem ser usados com $_GET.
https://www.alura.com.br/artigos/rest-conceito-e-fundamentos